partialWaveFit/fitResult.h

```diff
@@ -583,7 +583,7 @@ namespace rpwa {
 			         << "cannot deduce corresponding wave name." << std::endl;
 			return "";
 		}
-		return prodAmpName(3, prodAmpName.Length() - 3);
+		return prodAmpName(3, prodAmpName.Length() - 3).Data();
 	}
```

Comment:
this function `waveNameForProdAmp` should probably entirely be rewritten taking into account multi-digit ranks.